ner4opt/__init__.py

```diff
@@ -0,0 +1,24 @@
+"""
+:Version: 1.0.0 of April 9, 2023
+This module defines the public interface of the
+**Ner4Opt Library** providing access to the following modules:
+    - ``Ner4Opt``
+
+Example Usage
```

Comment:
See my changes in the README example and duplicate it here

Reply:
BTW, I think we should remove this from here and move it under the ner4opt.py file below the pydoc of Ner4Opt, so taht anyone looking for the code documentation in the main Ner4Opt class will see it

Reply:
I moved it to the main class.